### PR TITLE
cards

### PR DESCRIPTION
**Pull request by saschahaselhuhn**
🐛 Move Fonts in public folder, 

🚧 Add custom fonts to button

✨ Add Card-Function - Card.tsx

🐛 fixing wrong code typing

✨ Build Cards.stories.tsx with dummy data

✨ Build cards.module.css with dummy data

💄 Add example styling to Card

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -35,3 +36,16 @@ a {
...
+  /* root colors */
+  --MainClr: #890000;
+  --PrimaryClr: white;
+  --InputBgClr: #f2f2f2;
+  --InputTxtClr: black;
+
+  /* root fonts */
+  --BodyFont: "TrajanProRegular", sans-serif;
+  --BodyBoldFont: "TrajanProBold", sans-serif;
+  --StyleFont: "SilliiwillinnRegular", sans-serif;
```
Comment (phlgr) [inline styles/globals.css:53]: PascalCase is pretty unusual for naming CSS variables.
Also don't try to use abbreviations.

File: components/button/Button.module.css [hunk 2]
```diff
@@ -1,8 +1,10 @@
 .btn {
   border-radius: 10px;
+  font-family: var(--BodyFont);
+  font-size: 30px;
```
Comment (phlgr) [inline components/button/Button.module.css]: These should be responsive

File: components/cards/Cards.module.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.recipecard {
```
Comment (phlgr) [inline components/cards/Cards.module.css]: As you are using the `Cards.module.css` it is already clear that you are describing classes for the cards component.
```suggestion
.recipe {
```

File: components/cards/Cards.module.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  border: 3px solid var(--MainClr);
+  border-radius: 15px;
+  padding: 5px;
+  width: 80vw;
+  background-color: rgba(255, 255, 255, 0.75);
+  -webkit-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  -moz-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
```
Comment (phlgr) [inline components/cards/Cards.module.css]: You don't need these as these get generated automatically when building.

File: components/cards/Cards.module.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  gap: 0 0;
+  grid-template-areas:
+    "Img Img Name Name Name"
+    "Img Img Name Name Name"
+    "Img Img Name Name Name"
+    "Img Img Button Button Button";
+}
+
+.recipecardimgdiv {
```
Comment (phlgr) [inline components/cards/Cards.module.css]: You should use camelCase, also this naming is very complex.
Again the Card in the class name is not necessary.

File: components/cards/Cards.module.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  text-align: right;
+}
+
+.recipecardimgdiv img {
+  width: 100%;
+  border-radius: 50%;
+  border: 0.4em solid var(--MainClr);
+  margin-bottom: 0;
+  -webkit-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  -moz-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
```
Comment (phlgr) [inline components/cards/Cards.module.css]:
```suggestion
```

File: components/cards/Cards.module.css [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  justify-content: center;
+  align-items: center;
+  font-size: 1em;
+}
+.recipecardopendiv {
+  grid-area: Button;
+  text-align: right;
+}
+
+.recipecardimgdiv img {
```
Comment (phlgr) [inline components/cards/Cards.module.css]: This selector doesn't look to intuitive

File: components/cards/Cards.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  component: Cards,
+} as Meta;
+
+const Template: Story<Props> = (args) => <Cards {...args} />;
+
+export const RecipeCard = Template.bind({});
+RecipeCard.args = {
+  cardrecipesrc: "./images/foodimages/paellamallorquina.jpg",
+  cardrecipename: "Paella Mallorquina",
+  cardopenrecipesrc: "./images/go.svg",
```
Comment (phlgr) [inline components/cards/Cards.stories.tsx]: Those should also be in camelCase and the naming is a bit overcomplicated

File: components/cards/Cards.tsx [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+export type Props = {
+  cardrecipesrc: string;
+  cardrecipename: string;
+  cardopenrecipesrc: string;
+};
+
+function RecipeCard({
+  cardrecipesrc,
+  cardrecipename,
+  cardopenrecipesrc,
```
Comment (phlgr) [inline components/cards/Cards.tsx]: This naming would be clearer, also removed the prop for the button imgSrc as this doesn't need to change this often.
```suggestion
  imgSrc,
  name,
```

File: components/cards/Cards.module.css [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+  align-items: center;
+}
+.recipecardnamediv {
+  grid-area: Name;
+  font-family: var(--StyleFont);
+  color: var(--MainClr);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1em;
```
Comment (phlgr) [inline components/cards/Cards.module.css]: `font-size` is 1em by default
```suggestion
```

File: components/cards/Cards.module.css [hunk 11]
```diff
@@ -0,0 +1,52 @@
+.recipecard {
+  border: 3px solid var(--MainClr);
+  border-radius: 15px;
+  padding: 5px;
+  width: 80vw;
```
Comment (phlgr) [inline components/cards/Cards.module.css]: I wouldn't use these absolute values here, just use the `vw` and `vh` properties very carefully

File: components/cards/Cards.module.css [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+  background-color: rgba(255, 255, 255, 0.75);
+  -webkit-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  -moz-box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.8);
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  gap: 0 0;
```
Comment (phlgr) [inline components/cards/Cards.module.css]:
```suggestion
```

File: components/cards/Cards.module.css [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+  grid-template-areas:
+    "Img Img Name Name Name"
+    "Img Img Name Name Name"
+    "Img Img Name Name Name"
+    "Img Img Button Button Button";
+}
```
Comment (phlgr) [inline components/cards/Cards.module.css]: This seems a bit overcomplicated

**Review by phlgr — CHANGES_REQUESTED** [review]

File: components/cards/Cards.module.css (hunk 12 above)
Comment (saschahaselhuhn) [reply components/cards/Cards.module.css]: would be fixed in next pullrequest. thank you

File: styles/globals.css (hunk 1 above)
Comment (saschahaselhuhn) [reply styles/globals.css:53]: pascal-case is my coding style... its unusual, but is it bad? greetings

File: components/button/Button.module.css (hunk 2 above)
Comment (saschahaselhuhn) [reply components/button/Button.module.css]: would be fixed in next pullrequest. thank you

File: components/cards/Cards.module.css (hunk 3 above)
Comment (saschahaselhuhn) [reply components/cards/Cards.module.css]: would be fixed in next pullrequest. thank you

File: components/cards/Cards.module.css (hunk 5 above)
Comment (saschahaselhuhn) [reply components/cards/Cards.module.css]: would be fixed in next pullrequest. thank you

File: components/cards/Cards.module.css (hunk 6 above)
Comment (saschahaselhuhn) [reply components/cards/Cards.module.css]: would be fixed in next pullrequest. thank you

**Pull request merged**